### DISCUSSION TITLE
[Announcement] Final exam is graded & claim session

### DISCUSSION THREAD

**Discussion by KAIST-JongchanPark** [prosyslab-classroom/cs524-program-analysis#315] [Announcements]
Hi all,

Final exam is graded.

You can check your grade in Gradescope and KLMS. We made a assignment "final", so you can check out your personal score there. 

In Gradescope, you can check detail score criteria and feedback.

------------------------

For final exam, we have a claim session, 12/14, 15 (Wed, Thu).

You can claim via e-mail, or in person (by appointment).

Feel free to contact us if you have any questions. ([cs524.ta@prosys.kr](mailto:cs524.ta@prosys.kr))

Regards,
Jongchan

**Comment by PSuHyeon** [comment]
Can we know any statistics of exam grade? (for example, standard deviation)

**Comment by KihongHeo** [comment]
![스크린샷 2022-12-13 오후 7 50 00](https://user-images.githubusercontent.com/1281633/207298127-f6fbc699-1cfa-4cf8-b1d1-f268c6e76891.png)
